go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/zawachte/lomba
 
-go 1.19
+go 1.18
```

Comment:
This is modified due to my system having go v1.18, I am working on upgrading it.